lib/index.js

```diff
@@ -248,9 +248,18 @@ CallsiteRecord.fromStackFrames = function (stackFrames, fnName, typeName) {
     return null;
 };
 
-CallsiteRecord.fromError = function (error) {
+CallsiteRecord.fromError = function (error, topFrameFilter) {
```

Comment:
such a mess with this name...

"Okay, so I have a stack. But I can immediately say which frame is on top. Why am I suppose to provide the `isTopFrame` callback?"

In this sense, `findTopFrame` or even `selectTopFrame` look better. But they are more like method names...